XU-TechnicalPaper.md

```diff
@@ -367,7 +387,7 @@ Flawed incentivisation is believed to be the main reason why other solutions for
 - A user has X trading volume on Exchange Union, this user gets rewarded with XUC from his exchange platform
 - This is a program of each individual exchange
 
-*By default, each XUD opens a XUC payment channel to be able to pay makers, for order book relays and other services and also receive XUC payments for these services.*
+*By default, each XUD opens a XUC payment channel to be able to pay makers, for order book relays and other services and also receive XUC payments for these services. A missing XUC channel with a certain bi-directional volume results in a error message upon calling the `placeOrder(params)` API.*
```

Comment:
What do you mean by "certain bi-directional volume"? At the end, I'd change it to "calling the `placeOrder` API method."

Reply:
Fixed.